tgui/packages/tgui/interfaces/CentcomPodLauncher.js

```diff
@@ -1,518 +1,1125 @@
+          selected={data.effectName}
+          tooltip={multiline`
+      Edit pod's
+      name/desc.`}
+          tooltipPosition="bottom-left"
+          onClick={() => act('effectName')} />
+      )}>
+      {STYLES.map((page, i) => (
+        <Button
+          key={i}
+          width="45px"
+          height="45px"
+          tooltipPosition={
+            i >= STYLES.length-2
+              ? (i%2===1 ? "top-left" : "top-right")
+              : (i%2===1 ? "bottom-left" : "bottom-right")
+          }
+          tooltip={page.title}
+          style={{
+            'vertical-align': 'middle',
+            'margin-right': '5px',
+            'border-radius': '20px',
+          }}
+          selected={data.styleChoice-1 === i}
+          onClick={() => act('setStyle', { style: i })}>
+          <Box
+            className={classes(['supplypods64x64', 'pod_asset'+(i+1)])}
+            style={{
+              'transform': 'rotate(45deg) translate(-25%,-10%)' }} />
+        </Button>
+      ))}
+    </Section>
+  );
+};
+
+const Bays = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [compact, toggleCompact] = useLocalState(context, 'compact', 0);
+  return (
+    <Section title={compact ? "B" : "Bay"} fill buttons={(
+      <Fragment>
+        <Button
+          icon="trash"
+          color="transparent"
+          tooltip={multiline`
+        Clears everything
+        from the selected bay`}
+          tooltipOverrideLong
+          tooltipPosition="bottom-right"
+          onClick={() => act('clearBay')} />
+        <Button
+          icon="question"
+          color="transparent"
+          tooltip={multiline`
+      Each option corresponds
+      to an area on centcom.
+      Launched pods will
+      be filled with items
+      in these areas according
+      to the "Load from Bay"
+      options at the top left.`}
+          tooltipOverrideLong
+          tooltipPosition="bottom-right" />
+      </Fragment>
+    )}>
+      {BAYS.map((bay, i) => (
+        <Button
+          key={i}
+          content={bay.title}
+          tooltipPosition={"bottom-right"}
+          selected={data.bayNumber === i+1}
+          onClick={() => act('switchBay', { bayNumber: i+1 })} />
+      ))}
+    </Section>
+  );
+};
+
+const SyncTimerValues = context => {
+  const { act, data } = useBackend(context);
+  const [
+    quickTimerValues,
+    setTimerValues,
+  ] = useLocalState(context, 'quickTimerValues');
+  setTimerValues([
+    data["delay_1"]/10,
+    data["delay_2"]/10,
+    data["delay_3"]/10,
+    data["delay_4"]/10,
+  ]
+  );
+};
+
+const Timing = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [
+    compact,
+    toggleCompact,
+  ] = useLocalState(context, 'compact', 0);
+  return (
+    <Section title="Delay" fill buttons={(
+      <Button
+        icon="undo"
+        color="transparent"
+        tooltip={multiline`
+      Reset all pod
+      timings/delays`}
```

Comment:
Yeah I think these things do not autoformat well, so please fix indentation manually everywhere you can, like this:

```suggestion
        tooltip={multiline`
          Reset all pod
          timings/delays`}
```